chunky/src/java/se/llbit/chunky/main/Chunky.java

```diff
@@ -405,23 +395,35 @@ public SceneFactory getSceneFactory() {
   }
 
   @PluginApi
-  public void setPreviewRayTracerFactory(RayTracerFactory previewRayTracerFactory) {
-    this.previewRayTracerFactory = previewRayTracerFactory;
+  public void addPreviewRenderer(String name, Renderer renderer) {
+    if (InternalRenderManager.previewRenderers.containsKey(name))
+      Log.error("Preview renderer already exists (do you have the same plugin loaded twice?): " + name);
+    InternalRenderManager.previewRenderers.computeIfAbsent(name, n -> renderer);
```

Comment:
If the same plugin is loaded twice, then better take the _last_ registered renderer, not the first one so that we don't have weird classpath problems